README.md

```diff
@@ -116,6 +116,13 @@ To enable S3 server-side encryption, set to the algorithm to use when storing th
 
 If using S3 server-side encryption with `aws:kms`, the KMS Key ID to use for object encryption.
 
+* `WALG_CSE_KMS_ID`
```

Comment:
These settings, they are S3-only, aren't they? If they are, I think we should use WALG_S3_CSE_...

Reply:
In most cases you want to use it with s3, but they can be used with any storage backend (you can use AWS KMS encryption and GCP storage)